### ISSUE TITLE
Bug: Kama indicator definition not found

### ISSUE THREAD

**Issue by only4lee** [DaveSkender/Stock.Indicators#275]
I love your highly useful indicators project -- thank you for developing and sharing this. Amazing.

In any event, I'm running into this issue with both v1.2 and v1.3 NuGet packages:

I'm using a number of indicators such as GetStoch(), GetRsi(), etc. without issue. However, GetKama() is not present.

Brief example:
```
using Skender.Stock.Indicators;
...
... blah blah etc. etc.
...
... {
                var quoteHistory = tradeRepo.FindAllAsQuotes($"{stockName}_historic");
                
                var stoResults = Indicator.GetStoch(quoteHistory);   // works no issues
                var rsiResults = Indicator.GetRsi(quoteHistory); // works no issues
                var kemaResults = Indicator.GetKama(quoteHistory); // GetKama() not found
... }
```

Build result:

Error CS0117 'Indicator' does not contain a definition for 'GetKama'

**Comment by DaveSkender** [comment]
We just merged the code for KAMA last week, but have not yet delivered the new package to the public NuGet repository.  Unfortunately, that means the documentation is slightly ahead of the actually released package.

KAMA will be available in version 1.4.0.  I'm trying to get a few other indicators bundled in before releasing.  If you're urgently needing KAMA, MAMA, or ALMA, you can get the (beta) pre-release packages from our internal NuGet repo: [here](https://dev.azure.com/skender/Stock.Indicators/_packaging?_a=package&feed=Internal&package=Skender.Stock.Indicators&protocolType=NuGet).

I'm hoping to get v1.4.0 released later tonight, but it could be delayed if I run into troubles.

**Comment by only4lee** [comment]
Understood, thank you for the quick response. I'm doing a small project with my son using your library and am having a blast. If for some reason we actually have a positive outcome, my son says we owe you a coke :)
